spark/src/main/kotlin/com/adevinta/spark/components/divider/Divider.kt

```diff
@@ -21,92 +21,215 @@
+        }
+
+        MaterialVerticalDivider(
+            modifier = topModifier,
+            color = intent.color(),
+        )
+
+        if (!label.isNullOrBlank()) {
+            Text(
+                text = label,
+                style = SparkTheme.typography.body1,
+                textAlign = TextAlign.Center,
+                modifier = Modifier.padding(vertical = 16.dp),
+            )
+        }
+
+        MaterialVerticalDivider(
+            modifier = bottomModifier,
+            color = intent.color(),
+        )
+    }
 }
 
 @Preview(
-    group = "Separator",
+    group = "Dividers",
     name = "Divider",
 )
 @Composable
 internal fun DividerPreview(
     @PreviewParameter(ThemeProvider::class) theme: ThemeVariant,
 ) {
     PreviewTheme(theme) {
-        Divider()
-        Divider()
+        HorizontalDivider(intent = DividerIntent.Outline)
+        HorizontalDivider(intent = DividerIntent.OutlineHigh)
+        HorizontalDivider(intent = DividerIntent.OutlineHigh, label = "Label")
+        HorizontalDivider(
+            intent = DividerIntent.Outline,
+            label = "Label",
+            labelHorizontalAlignment = LabelHorizontalAlignment.Start,
+        )
+        HorizontalDivider(
+            intent = DividerIntent.OutlineHigh,
+            label = "Label",
+            labelHorizontalAlignment = LabelHorizontalAlignment.End,
+        )
+
+        Row {
+            VerticalDivider(
+                intent = DividerIntent.OutlineHigh,
+                label = "Label",
+                labelVerticalAlignment = LabelVerticalAlignment.Top,
+            )
+            VerticalDivider(
+                intent = DividerIntent.OutlineHigh,
+                label = "Label",
+                labelVerticalAlignment = LabelVerticalAlignment.Center,
+            )
+            VerticalDivider(
+                intent = DividerIntent.OutlineHigh,
+                label = "Label",
+                labelVerticalAlignment = LabelVerticalAlignment.Bottom,
+            )
+            HorizontalSpacer(space = 16.dp)
+            VerticalDivider(
+                labelVerticalAlignment = LabelVerticalAlignment.Bottom,
+            )
+            HorizontalSpacer(space = 16.dp)
+
+            VerticalDivider(
+                intent = DividerIntent.OutlineHigh,
+                labelVerticalAlignment = LabelVerticalAlignment.Bottom,
+            )
+            HorizontalSpacer(space = 16.dp)
+
+            VerticalDivider(
+                label = "Label",
+                labelVerticalAlignment = LabelVerticalAlignment.Top,
+            )
+            VerticalDivider(
+                label = "Label",
+                labelVerticalAlignment = LabelVerticalAlignment.Center,
+            )
+            VerticalDivider(
+                label = "Label",
+                labelVerticalAlignment = LabelVerticalAlignment.Bottom,
+            )
+        }
     }
 }
+
+@Composable
+public fun ColumnScope.Sample() {
```

Comment:
This doesn't seem to be used anywhere? We must remove it since it's also public and will pollute users autocomplete